src/commands/Nuxeo/PreviewCommand.ts

```diff
@@ -0,0 +1,86 @@
+import debug from 'debug';
+import { Arguments, Argv, CommandModule } from 'yargs';
+import { ProcessSpawner } from '../../lib/ProcessSpawner';
+
+const log: debug.IDebugger = debug('command:nuxeo:preview');
+
+/**
+ * Nuxeo Preview Command
+ */
+export class PreviewCommand implements CommandModule {
+  public command: string = 'preview';
+
+  public describe: string = 'Run a preview based on presets';
+
+  public builder: (args: Argv) => Argv = (args: Argv) => {
+    args.option({
+      'no-comment': {
```

Comment:
Either we really want to limit possible opts passed to `jx`, or, as we seen with`jx step helm` is really annoying to not have all the possible argument available.
If you think that it makes sence to pass any opts to `jx`, might be more interesting to rely on https://github.com/yargs/yargs/blob/master/docs/examples.md#and-non-hyphenated-options-too-just-use-argv_ and pass every opts to `jx` sub commands; plus a few default values with the `{...XXX}` TS syntax (same than `Object#assign` is ES)